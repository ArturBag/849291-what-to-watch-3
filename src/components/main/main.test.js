import React from "react";
import renderer from "react-test-renderer";
import Main from "./main.jsx";

const filmsList = [
  {
    title: `Breaking Bad`,
    genre: `Thriller`,
    issuedDate: 2000,
    preview: `../../markup/img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `Pirates`,
    genre: `Drama`,
    issuedDate: 1999,
    preview: `../../markup/img/bohemian-rhapsody.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `Misfits`,
    genre: `Drama`,
    issuedDate: 1998,
    preview: `../../markup/img/macbeth.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `Terminal`,
    genre: `Horror`,
    issuedDate: 1992,
    preview: `../../markup/img/aviator.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `Star Wars`,
    genre: `Documentary`,
    issuedDate: 2002,
    preview: `../../markup/img/we-need-to-talk-about-kevin.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `Forsage`,
    genre: `Crime`,
    issuedDate: 2001,
    preview: `../../markup/img/what-we-do-in-the-shadows.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `Goal`,
    genre: `Comedy`,
    issuedDate: 2011,
    preview: `../../markup/img/orlando.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  },
  {
    title: `New Yorker`,
    genre: `History`,
    issuedDate: 2010,
    preview: `../../markup/img/war-of-the-worlds.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
  ultrices ut elit.`,
    rating: 2.2,
    director: `Angel Calderon`,
    starring: [`Linda Horne, Angel Calderon`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  }
];

const MovieHeader = {
  TITLE: filmsList[0].title,
  GENRE: filmsList[0].genre,
  ISSUED_DATE: filmsList[0].issuedDate
};

it(`Render Main`, () => {

  const tree = renderer
    .create(<Main
      title={MovieHeader.TITLE}
      genre={MovieHeader.GENRE}
      issuedDate={MovieHeader.ISSUED_DATE}
      moviesList={filmsList}
      onMovieCardTitleClick={() => { }}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();

});

