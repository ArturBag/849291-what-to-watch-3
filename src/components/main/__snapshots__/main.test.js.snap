// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Main 1`] = `
Array [
  <section
    className="movie-card"
  >
    <div
      className="movie-card__bg"
    >
      <img
        alt="The Grand Budapest Hotel"
        src="img/bg-the-grand-budapest-hotel.jpg"
      />
    </div>
    <h1
      className="visually-hidden"
    >
      WTW
    </h1>
    <header
      className="page-header movie-card__head"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="user-block"
      >
        <div
          className="user-block__avatar"
        >
          <img
            alt="User avatar"
            height={63}
            src="img/avatar.jpg"
            width={63}
          />
        </div>
      </div>
    </header>
    <div
      className="movie-card__wrap"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster"
        >
          <img
            alt="The Grand Budapest Hotel poster"
            height={327}
            src="img/the-grand-budapest-hotel-poster.jpg"
            width={218}
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            Breaking Bad
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Thriller
            </span>
            <span
              className="movie-card__year"
            >
              2000
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              type="button"
            >
              <svg
                height={19}
                viewBox="0 0 19 19"
                width={19}
              >
                <use
                  xlinkHref="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              type="button"
            >
              <svg
                height={20}
                viewBox="0 0 19 20"
                width={19}
              >
                <use
                  xlinkHref="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <section
      className="catalog"
    >
      <GenresList
        activeItem={null}
        getActiveItem={[Function]}
        moviesList={
          Array [
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Bernard Harding",
              "genre": "Thriller",
              "issuedDate": 2000,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/fantastic-beasts-the-crimes-of-grindelwald.jpg",
              "rating": 8.9,
              "ratingCount": 59,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "runTime": Object {
                "hours": 1,
                "minutes": 22,
              },
              "starring": Array [
                "Bill Murray, Edward Norton, Jude Law, Willem Dafoe and other",
              ],
              "title": "Breaking Bad",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Will Brooks",
              "genre": "Drama",
              "issuedDate": 1999,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/bohemian-rhapsody.jpg",
              "rating": 4.5,
              "ratingCount": 11,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Kymani Burgess, Jaylin Cherry, Jacey Oliver, Conrad Wolf and other",
              ],
              "title": "Pirates",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Titus Sparks",
              "genre": "Drama",
              "issuedDate": 1998,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/macbeth.jpg",
              "rating": 6.7,
              "ratingCount": 24,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Giovanny Mccullough, Leah Delacruz, Davon Terrell, Romeo Carr and other",
              ],
              "title": "Misfits",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Jeramiah Goldeng",
              "genre": "Horror",
              "issuedDate": 1992,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/aviator.jpg",
              "rating": 6.9,
              "ratingCount": 555,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Nasir Castaneda, Ariel Brooks, Lucia Compton and other",
              ],
              "title": "Terminal",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Roy Boyle",
              "genre": "Documentary",
              "issuedDate": 2002,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/we-need-to-talk-about-kevin.jpg",
              "rating": 8.3,
              "ratingCount": 242,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Molly Reyes, Edward Norton, Mareli Moore, Makenna Henderson and other",
              ],
              "title": "Star Wars",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Claire Hodge",
              "genre": "Crime",
              "issuedDate": 2001,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/what-we-do-in-the-shadows.jpg",
              "rating": 2.5,
              "ratingCount": 243,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Amaris Hess, Tobias Osborne, Samir Fitzpatrick, Baron Bolton and other",
              ],
              "title": "Forsage",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Brynlee Haney",
              "genre": "Comedy",
              "issuedDate": 2011,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/orlando.jpg",
              "rating": 3.4,
              "ratingCount": 222,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Piper Mccann, Seamus Ashley, Jude Law, Lila Waters and other",
              ],
              "title": "Goal",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
            Object {
              "background": "img/bg-the-grand-budapest-hotel.jpg",
              "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.",
              "director": "Melany Short",
              "genre": "History",
              "issuedDate": 2010,
              "poster": "img/the-grand-budapest-hotel-poster.jpg",
              "preview": "../../markup/img/war-of-the-worlds.jpg",
              "rating": 2.9,
              "ratingCount": 442,
              "reviews": Array [
                Object {
                  "date": "December 24, 2016",
                  "name": "Kate Muir",
                  "reviewText": "Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.",
                  "score": 2.9,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Bill Goodykoontz",
                  "reviewText": "Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        \\"The Grand Budapest Hotel\\" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.",
                  "score": 8.2,
                },
                Object {
                  "date": "November 18, 2015",
                  "name": "Amanda Greever",
                  "reviewText": "I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.",
                  "score": 8.4,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Matthew Lickona",
                  "reviewText": "The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.",
                  "score": 6.2,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6.6,
                },
                Object {
                  "date": "December 20, 2016",
                  "name": "Paula Fleri-Soler",
                  "reviewText": "It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.",
                  "score": 6,
                },
              ],
              "starring": Array [
                "Franklin Sosa, Abby Beasley, Jovan Lang and other",
              ],
              "title": "New Yorker",
              "videoSrc": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
            },
          ]
        }
        onMovieCardTitleClick={[Function]}
      />
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          © 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
